### PR TITLE
Feature/46573 rest api endpoint for zaak instance link

### PR DESCRIPTION
**Pull request by brunofarroporitense**
[Story](https://ritense.tpondemand.com/restui/board.aspx?#page=userstory/46573)

Small note that i had some issues with the mapping of the id property, since jackson checks all the getters and setters. I think i'm sending the correct id, but if you guys could double check! Thanks!

### CODE REVIEW

File: openzaak/src/main/kotlin/com/ritense/openzaak/exception/ZaakInstanceLinkNotFoundException.kt [hunk 1]
```diff
@@ -1,3 +1,16 @@
 package com.ritense.openzaak.exception
```
Comment (Klaas-Ritense) [inline openzaak/src/main/kotlin/com/ritense/openzaak/exception/ZaakInstanceLinkNotFoundException.kt:1]: This file still misses the Copyright header. Could you add it?

File: zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt [hunk 2]
```diff
@@ -62,4 +65,21 @@ class ZaakObjectResource(
         val form = zaakObjectService.getZaakObjectForm(objectUrl)
         return form?.let { ResponseEntity.ok(it) } ?: ResponseEntity.notFound().build()
     }
+
+    @GetMapping(value = ["/v1/zaakinstancelink/zaak"])
```
Comment (Klaas-Ritense) [inline zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt:69]: All endpoints should be added to a `...HttpSecurityConfigurer`

File: zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt [hunk 3]
```diff
@@ -62,4 +65,21 @@ class ZaakObjectResource(
...
+    fun getZaakInstanceLink(
+        @RequestParam(name = "zaakInstanceUrl") zaakInstanceUrl: URI
+    ): ResponseEntity<ZaakInstanceLink>{
```
Comment (Klaas-Ritense) [inline zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt:72]: The responses of both endpoints, contain more data than they should. I suggest to respond with a DTO

Comment (brunofarroporitense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt:72]: Should i just return the "zaakInstanceLinkId", "zaakInstanceUrl", "zaakInstanceId", "documentId" and "zaakTypeUrl"? or is this still too much information?

Comment (Klaas-Ritense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt:72]: A `zaakInstanceUrl` and `documentId` is enough

Comment (brunofarroporitense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt:72]: Alright, thank you

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**